### DISCUSSION TITLE
Can DataTime data be inserted directly into UnixTime data?

### DISCUSSION THREAD

**Discussion by chinaxushi** [apache/doris#29970] [A - General / Q&A]
datatime类型数据可以直接插入unixtime的数字嘛。在2.0.3版本中新建了一个测试表，两个字段，其中一个是datatime类型数据。我尝试直接插入unixtime的数字，提示插入成功。但查看的时候，看到内容都是NULL

**Comment by LemonLiTree** [comment]
不行 用函数转
